### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by PhoenixCodes**
Changed attackButton() to be selectable - pass it true or false to turn it on or off, or leave it nil to toggle.
Changed initial battlerage parsing to spam send(" ") less and still work perfectly.

### CODE REVIEW

File: script.lua [hunk 1]
```diff
@@ -881,14 +878,27 @@ keneanung.bashing.flee = function()
 	kecho("New order. Tactical retreat.\n")
 end
 
-keneanung.bashing.attackButton = function()
-	if keneanung.bashing.attacking == 0 then
+keneanung.bashing.attackButton = function(toggle)
+    if toggle == nil then
+		if keneanung.bashing.attacking == 0 then
+			if keneanung.bashing.setTarget() then
```
Comment (keneanung) [inline script.lua]: Can we agree to avoid this code duplication?

File: script.lua [hunk 2]
```diff
@@ -881,14 +878,27 @@ keneanung.bashing.flee = function()
...
-		startAttack()
-		kecho("Nothing will stand in our way.\n")
-                else
-                kecho("Nothing to target, boss.\n")
-                end
+			startAttack()
+			kecho("Nothing will stand in our way.\n")
+        else
+            kecho("Nothing to target, boss.\n")
+        end
```
Comment (keneanung) [inline script.lua]: There is some whitespace at the end of this line.

**Comment by PhoenixCodes** [comment]
Yeah, I should be able to trim that out.

Roland Goldberg
----

**Comment by PhoenixCodes** [comment]
That should take care of it!

**Pull request merged**